BeeKit/HeathKit/HealthKitConfig.swift

```diff
@@ -51,6 +51,10 @@ public enum HealthKitConfig {
             QuantityHealthKitMetric(humanText: "Vitamin K", databaseString: "dietaryVitaminK", category: .Nutrition, hkQuantityTypeIdentifier: .dietaryVitaminK),
             QuantityHealthKitMetric(humanText: "Water", databaseString: "water", category: .Nutrition, hkQuantityTypeIdentifier: .dietaryWater),
             
+            // Self care
+            ToothbrushingDailyMinutesHealthKitMetric.make(),
+            ToothbrushingDailySessionsHealthKitMetric.make(),
```

Comment:
I don't want to have multiple top level metrics for the same underlying metric. There are lots of metrics where this could make sense, and the list risks getting far too long.
The way I think this should work:
1. There is one metric for toothbrushing.
2. On the preview/confirm page, there are additional settings to refine exactly how the metric is logged. These are stored using additional attributes on the autodata key on the goal. The preview updates as these settings are changed.
3. For toothbrushing in particular, this should be either "Aggregate Total Daily Minutes", or reporting each toothbrushing session as a separate data point with its number of minutes. People can count sessions via adjusting aggday.

Note this will require some tweaks to the sync code to allow multiple data points per day to be synced correctly. A while back the app started including requestid with these points to help facilitate this.

Reply:
Customizing `aggday` is possible with a custom goal. A typical do more (hustler) goal type will have its _how to aggregate the day_ set to sum.